### PR TITLE
updated the lines

### PR DESCRIPTION
**Pull request by ryuusama09**
Fixes #0000 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

File: app/views/users/profile.html.erb [hunk 1]
```diff
@@ -14,8 +14,9 @@
   <br />
 
   <div style="text-align: center;">
-  <h5><a href='/notes/author/<%= @profile_user.name %>'><%= @profile_user.note_count %><%= @profile_user.note_count == 1 ? ' research note' : ' research notes' %></a>
-  <a class="ml-2" href = "/tag/question:*/author/<%= params[:id] %>"><%= Node.questions.where(status: 1, uid: @profile_user.id).length %> questions </a></h5>
+  <h5><a href='/notes/author/<%= @profile_user.name %>'><%= pluralize(@profile_user.note_count, 'research note', plural: 'research notes') %></a>
+  +  <a class="ml-2" href = "/tag/question:*/author/<%= params[:id] %>"><%= pluralize(Node.questions.where(status: 1, uid: @profile_user.id).length, 'question')
```
Comment (TildaDares) [inline app/views/users/profile.html.erb:18]:
```suggestion
  <a class="ml-2" href = "/tag/question:*/author/<%= params[:id] %>"><%= pluralize(Node.questions.where(status: 1, uid: @profile_user.id).length, 'question')
```

**Review by TildaDares — CHANGES_REQUESTED** [review]
Feel free to just click the `Commit suggestion` button @ryuusama09.

**Comment by ryuusama09** [comment]
> Feel free to just click the `Commit suggestion` button @ryuusama09.

Implemented it  !

**Review by TildaDares — APPROVED** [review]
Great job @ryuusama09. Thank you!!

**Comment by TildaDares** [comment]
Congratulation on merging your PR @ryuusama09 🎉🎉

**Pull request merged**